### ISSUE TITLE
Travis has been failing `./factor -run=zealot.cli-changed-vocabs | paste -s -d " " -` recently

### ISSUE THREAD

**Issue by catb0t** [factor/factor#2050]
All the recent pull requests have failed the Linux CI with the message:

```
The command "./factor -run=help.lint `./factor -run=zealot.cli-changed-vocabs | paste -s -d " " -`" exited with 1.
```

but without any error from Factor. 

Search this full build log or any other recent Linux one for `exited with 1`: https://api.travis-ci.org/v3/job/417495274/log.txt

**Comment by catb0t** [comment]
```
==== FAILING LINT
==== resource:basis/furnace/auth/auth-docs.factor
resource:basis/furnace/auth/auth-docs.factor: 115
Asset: T{ link f "furnace.auth.providers" }
Help article does not exist
name "furnace.auth.providers.null"
resource:basis/furnace/auth/auth-docs.factor: 126
Asset: T{ link f "furnace.auth.features" }
Help article does not exist
name "furnace.auth.features.deactivate-user"
resource:basis/furnace/auth/auth-docs.factor: 135
Asset: T{ link f "furnace.auth.realms" }
Help article does not exist
name "furnace.auth.basic"
resource:basis/furnace/auth/auth-docs.factor: 181
Asset: T{ link f "furnace.auth" }
Help article does not exist
name "webapps.user-admin"
The command "./factor -run=help.lint `./factor -run=zealot.cli-changed-vocabs | paste -s -d " " -`" exited with 1.
```
https://travis-ci.org/factor/factor/jobs/417739831

**Comment by catb0t** [comment]
This isn't a travis thing, I can reproduce it locally

**Comment by catb0t** [comment]
This doesn't happen outside of Travis by the way

**Comment by mrjbq7** [comment]
Zealot is new way to test only affected vocabs to allow Travis to work for us. Maybe needs some help! @erg?

**Comment by catb0t** [comment]
closed in favour of  #2057
